commit c2168994647e44ecebdb00d5baf05bf443a7c2a5 (refs/changes/81/581/2)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2016-10-03T22:14:34+00:00 (3 years ago)
    
    MORAY-364 Want a better filter parser for Moray
    Reviewed by: Patrick Mooney <patrick.mooney@joyent.com>
    Approved by: David Pacheco <dap@joyent.com>

diff --git a/lib/objects/common.js b/lib/objects/common.js
index 51575e8..fac5728 100644
--- a/lib/objects/common.js
+++ b/lib/objects/common.js
@@ -12,7 +12,7 @@ var util = require('util');
 var net = require('net');
 
 var assert = require('assert-plus');
-var filters = require('ldap-filter');
+var filters = require('moray-filter');
 var ip6addr = require('ip6addr');
 var LRU = require('lru-cache');
 var once = require('once');
diff --git a/package.json b/package.json
index bdd356d..351b210 100644
--- a/package.json
+++ b/package.json
@@ -15,7 +15,7 @@
         "deep-equal": "0.0.0",
         "fast": "0.3.6",
         "ip6addr": "0.1.1",
-        "ldap-filter": "0.2.4",
+        "moray-filter": "1.0.0",
         "libuuid": "0.1.3",
         "lru-cache": "2.5.0",
         "node-manatee": "git+https://github.com/joyent/node-manatee.git#152d3e1",
